docs/UserGuide.md

```diff
@@ -58,121 +68,131 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 </div>
 
-### Viewing help : `help`
+<!-- ### Viewing help : `help`
 
 Shows a message explaning how to access the help page.
 
 ![help message](images/helpMessage.png)
 
-Format: `help`
+Format: `help` -->
 
+### Creating a project: `project`
 
-### Adding a person: `add`
+Create a project to be tracked by the application.
 
-Adds a person to the address book.
-
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `project n/NAME d/DESCRIPTION [t/TAG]`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
-Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+Example: `project n/Momentum d/CS2103T Team Project t/impt`
+
+Result: Creates a project named “Momentum” with a description “CS2103T Team Project” and a tag "impt".
+
+### Updating a project: `edit`
+
+Update a project that has been previously created.
+
+Format: `edit PROJECT_ID n/NAME d/DESCRIPTION [t/TAG]`
```

Comment:
Should both name and description be optional fields here?

Reply:
Name is not optional right?

Reply:
I think it should be. Because it doesn't make sense for the user to have to enter something for the name when they are not intending to change it. I think we might have made a mistake when doing writing this section of the user guide. 

So the edit command will be like the addressbook's, where all fields except index are optional, but at least one of the optional fields must be provided.

Reply:
Fixed